readme.md

```diff
@@ -37,7 +37,16 @@ gulp.task('default', () =>
 
 ## API
 
-### rev()
+### rev([options])
+
+#### options
+
+##### forceRev
+
+Type: `string`<br>
+Default: `false`
+
+Override the `revHash` of the file. This can be used when you want the rev to run, but have predicitable results, and a manifest file.
```

Comment:
The description doesn't clearly enough describe what it does or why something would use it.